src/localization/de/messages.json

```diff
@@ -156,7 +156,7 @@
   "druid.guardian.suggestions.rage.wasted": "{0}% verschwendete Wut",
   "druid.guardian.suggestions.thrash.uptime": "{0}% Uptime",
   "druid.resto.suggestions.innervate.avgManaSaved": "Durchschnittliche gesparte Mana pro Einsatz",
-  "druid.resto.suggestions.innervate.selfCasts": "Wieviele Male, dass du Anregen an dich selbst eingesetzt hast",
+  "druid.resto.suggestions.innervate.selfCasts": "Wieviele Male, dass du Anregen an dir selbst eingesetzt hast",
```

Comment:
![image](https://user-images.githubusercontent.com/36763177/116792275-e73de200-aabf-11eb-849a-934676b20d69.png)

Der Kontext lässt sich in der englischen Version schlecht fassen, ohne das Beispiel zu sehen. Dadurch ist der Satz etwas unlogisch geraten.

Vielleicht `mal Anregen auf dich selbst eingesetzt`